tests/instruments/rohdeschwarz/test_fsl_with_device.py

```diff
@@ -0,0 +1,82 @@
+# from time import sleep
+
+import pytest
+from pymeasure.instruments.rohdeschwarz.fsseries import FSL
+
+pytest.skip("Only works with connected hardware", allow_module_level=True)
+RESOURCE = "TCPIP::141.20.43.231::INSTR"
```

Comment:
We have a fixture, which looks for a certain parameter as a resource name.
Only if this is present, tests with a device are run, see the [test documentation](https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments/tests.html#device-tests).

Reply:
I added the fixture and left the ip as a placeholder, hope that's alright like this!

Reply:
for what do you need the `connected_device_address = "TCPIP::000.00.00.000::INSTR"` line?

Placeholder for what?

I'm not sure, whether your line won't override the fixture, breaking the whole setup.